src/main/java/ch/uzh/ifi/hase/soprafs21/rest/dto/UserGetDTO.java

```diff
@@ -5,8 +5,8 @@
 public class UserGetDTO {
 
     private Long id;
-    private String username;
-    private UserStatus status;
+    private String email;
+    private UserStatus online;
```

Comment:
```suggestion
    private boolean online;
```
I like the naming `getOnline()` and `setOnline()`.  I'd suggest to also remove `UserStatus` completely to avoid confusion with the match status of a user and since we only need a status `ONLINE` or `OFFLINE`. Instead just use the `status` variable as a boolean? And at some point later maybe even as a date: What if someone just closes the website: with a boolean the user will stay logged in, with a lastOnline datetime field, we could automatically display the user offline if datetime+15min < datetime.now().

Reply:
Love it

Reply:
Done.